pos_order_wait_save/static/src/js/screens.js

```diff
@@ -0,0 +1,86 @@
+/* Copyright (C) 2020-Today: Druidoo (<https://www.druidoo.io>)
+   License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). */
+odoo.define('pos_order_wait_save.models', function (require) {
+    "use strict";
+    var screens = require('point_of_sale.screens');
+    var pos_model = require('point_of_sale.models');
+    var PaymentScreenWidget = screens.PaymentScreenWidget;
+    var OrderSuper = pos_model.Order.prototype;
+    pos_model.Order = pos_model.Order.extend({
+        initialize: function (attributes, options) {
+            OrderSuper.initialize.apply(this, arguments);
+            this.is_save_order = false;
+        },
+        export_as_JSON: function () {
+            var res = OrderSuper.export_as_JSON.apply(this, arguments);
+            res.is_save_order = this.is_save_order;
```

Comment:
```suggestion
            res.is_order_saved = this.is_order_saved;
```

Please rename this field everywhere

Reply:
Actually it looks like this field is true when the order is not saved yet.
So maybe you should rename it to `is_not_saved_yet`